README.md

```diff
@@ -27,10 +28,17 @@ On the target server:
      # git clone https://github.com/RedHatSatellite/satellite-clone.git
    ```
 
-2. Install `ansible` package.  Ansible should be installed from extras channel.
+2. Install the `ansible` package.
+   For Satellite 6.9 and newer:
+   ```console
+     # subscription-manager repos --enable rhel-7-server-ansible-2.9-rpms
+     # yum install ansible
+   ```
+
+   Otherwise:
    ```console
      # subscription-manager repos --enable rhel-7-server-extras-rpms
-     # yum install -y ansible
+     # yum install ansible
```

Comment:
Nobody should install ansible from Extras these days

Reply:
I'm going to update this with some sort of matrix (satversion, ansible version)

Reply:
@evgeni have a look now!